README.md

```diff
@@ -326,6 +329,18 @@ Simp::BeakerHelpers::DEFAULT_PUPPET_AGENT_VERSION, which is currently '1.10.4'.
 
 ## Environment variables
 
+#### `BEAKER_SIMP_webserve`
```

Comment:
Not a blocker, but why `webserve` instead of `webserver?`

Reply:
Just felt right at the time and doesn't make me think that I'm spinning up Apache or something.

I almost made it `webrick` explicitly. Would that be better?